test/cpp/mip/CMakeLists.txt

```diff
@@ -1,23 +1,31 @@
+# TODO: Disabled tests are old and broken. Should we keep these?
+
+#[[
 add_mip_test(
     NAME TestMipPerf
     SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/mip_parser_performance.cpp"
     COMMAND TestMipPerf
     USE_CPP TRUE
 )
+]]
 
+#[[
 add_mip_test(
     NAME TestMipCpp
     SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/test_mip.cpp"
     COMMAND TestMipCpp
     USE_CPP TRUE
 )
+]]
 
+#[[
 add_mip_test(
     NAME TestMipPacket
     SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/test_packet_interface.cpp"
     COMMAND TestMipPacket
     USE_CPP TRUE
 )
+]]
```

Comment:
Are all of these broken? They should still work and some would be good to run as if they were unit tests (i.e. verify that the MIP parser isn't completely broken).